src/ConsoleCommand.php

```diff
@@ -59,5 +65,21 @@ public function __unserialize(array $data): void
             objectOrMethod: $data['handler_class'],
             method: $data['handler_method'],
         );
+        $this->aliases = $data['aliases'];
+        $this->help = $data['help'];
+    }
+
+    /**
+     * @return ConsoleArgumentDefinition[]
+     */
+    public function getDefinition(): array
```

Comment:
This is nice! I'm just thinking about whether there can be a slightly better name. 

We have `ConsoleArgumentDefinition` for one specific argument, and then we have `getDefinition` for a collection of definitions. I feel that this method should maybe be called `getDefinitionList` or something alike to be more accurate. 

Another approach could be to have `ConsoleArgumentDefinition` and `ConsoleCommandDefinition`, but the latter one would just contain an array of argument definitions — I think. But maybe there are future features that might benefit from having a class to put command-specific meta data on. wdyt?

Reply:
That was great idea - makes things much clearer - thanks!